code/modules/mob/living/carbon/xenomorph/update_icons.dm

```diff
@@ -138,14 +139,47 @@
 
 	wound_overlay.vis_flags &= ~VIS_HIDE // Show the overlay
 
+/mob/living/carbon/xenomorph/proc/update_backpack()
+	if(!backpack_overlay)
```

Comment:
can you rename this to something like update_snowflake_overlays?

Reply:
done

Reply:
tivi why